package.build.ts

```diff
@@ -361,6 +361,20 @@ export const pkg = (modules: Builder.ParsedModule[]) => ({
               title: {
                 type: "string",
               },
+              menuType: {
+                anyOf: [
```

Comment:
`anyOf` is more of an "advanced" JSON Schema feature. In VS Code [`markdownEnumDescriptions`](https://code.visualstudio.com/api/references/contribution-points#Configuration-property-schema) is better supported.